source/common/upstream/subset_lb.cc

```diff
@@ -273,6 +274,26 @@ SubsetLoadBalancer::extractSubsetMetadata(const std::set<std::string>& subset_ke
   return kvs;
 }
 
+std::string SubsetLoadBalancer::describeMetadata(const SubsetLoadBalancer::SubsetMetadata& kvs) {
```

Comment:
Can we test this somehow? I think especially with https://github.com/envoyproxy/envoy/pull/2751 this won't even get run in most normal cases.

Reply:
Added a test case for this.